base/cvd/cuttlefish/host/libs/web/android_build_api.h

```diff
@@ -71,7 +71,7 @@ class BuildApi {
  public:
   BuildApi() = delete;
   BuildApi(BuildApi&&) = default;
```

Comment:
FYI this can now have some weird behaviors, e.g. creating a `BuildApi` from a `CachingBuildApi`

[Compiler explorer](https://godbolt.org/z/asbY83e61)

Reply:
Thanks for the catch!  Deleted the move constructors for both.  They were unused anyways.